docs/example-manifests/SLEMicro/dhcp/rke2-control-plane.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: sample-cluster
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+        - 192.168.0.0/18
+    services:
+      cidrBlocks:
+        - 10.96.0.0/12 # Doesn't matter because they are an internal network
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
+    kind: RKE2ControlPlane
+    name: sample-cluster
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: Metal3Cluster
+    name: sample-cluster
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: Metal3Cluster
+metadata:
+  name: sample-cluster
+  namespace: default
+spec:
+  controlPlaneEndpoint:
+    host: 192.168.125.200 # Has to match the IP of the control plane node
+    port: 6443
+  noCloudProvider: true
+---
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
+kind: RKE2ControlPlane
+metadata:
+  name: sample-cluster
+  namespace: default
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: Metal3MachineTemplate
+    name: sample-cluster-controlplane
+  replicas: 1
+  preRKE2Commands:
+    - /usr/lib/systemd/systemd-growfs /var   # this should be removed with https://bugzilla.suse.com/show_bug.cgi?id=1217430
+  agentConfig:
```

Comment:
It needs 'format: ignition' here otherwise the config drive will be created using cloud-init and it will fail.

Reply:
added. Thanks! good catch